src/components/common/StyledModal.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react'
+import { createGlobalStyle } from 'styled-components'
+import Modali, { ModalHook, toggleModaliComponent, useModali } from 'modali'
+
+import variables from 'styles/variables'
+import { MEDIA } from 'const'
+
+const ModaliGlobalStyle = createGlobalStyle`
+// global root variables
+  ${variables}
```

Comment:
Do we need this?

Reply:
i'll keep it inside the `global.scss` file as one of the few styles we keep there and remove it here :+1: